### PR TITLE
README Update for Evaluation

### PR DESCRIPTION
**Pull request by singankit**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ai/azure-ai-projects/README.md [hunk 1]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
 
 <!-- END SNIPPET -->
 
+### Evaluation
+Evaluation in Azure AI Project client library is desinged to assess the performance of generative AI applications in cloud. Generative AI application generations are quantitatively measured with mathematical based metrics, AI-assisted quality and safety metrics. Metrics are defined as evaluators. Built-in or custom evaluators can provide comprehensive insights into the application's capabilities and limitations.
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:722]: "in the cloud"

File: sdk/ai/azure-ai-projects/README.md [hunk 2]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
 
 <!-- END SNIPPET -->
 
+### Evaluation
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:721]: Space below title... MS markdown tools should call that out (I see squiggly lines under the title if there is space line below)

File: sdk/ai/azure-ai-projects/README.md (hunk 1 above)
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:722]: "Generative AI application generations" - generative / generations ... does not sound good to me.... Maybe say "The output of Generative AI application is quantitively..." or something similar

File: sdk/ai/azure-ai-projects/README.md [hunk 3]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
...
+#### Evaluator
+Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models or generative AI applications.
+
+Evaluators are made available via [azure-ai-evalaution][azure_ai_evaluation] SDK for local experience and also in [Evaluator Library][evaluator_library] in Azure AI Studio for using them in the cloud.
+
+More details on built-in and custom evaluators can be found [here][evaluators].
+
+#### Run Evalaution in cloud:
+
+To run evaluation in cloud following are needed:
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:733]: ... the following are needed:

File: sdk/ai/azure-ai-projects/README.md [hunk 4]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
...
+
+- Evaluators
+- Data to be evaluated
+- [Optional] Azure Open AI model.
+
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:738]: Can you say something about supporting non OpenAI models? Or is that too soon

File: sdk/ai/azure-ai-projects/README.md [hunk 5]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
...
+##### Evaluators
+
+For running evaluator in cloud, evaluator id is needed. To get via code you use [azure-ai-evaluation][azure_ai_evaluation]
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:741]: ID? (instead of id)

Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:741]: To get it via code

File: sdk/ai/azure-ai-projects/README.md [hunk 6]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
...
+```
+# pip install azure-ai-evaluation
+
+from azure.ai.evaluation import RelevanceEvalautor
+
+evaluator_id = RelevanceEvaluator.id
+```
+
+##### Data to be evaluated
+Evaluation in cloud supports data in form of `jsonl` file. Data can be uploaded via the helper method `upload_file` on the project client.
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:752]: in the cloud

File: sdk/ai/azure-ai-projects/README.md [hunk 7]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
...
+
+```python
+# Creating project client
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:760]: We have one place at the top where we define project_client. All samples can assume project_client is already defined, and no need to show it again.

File: sdk/ai/azure-ai-projects/README.md [hunk 8]
```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
...
+# Upload data for evaluation and get dataset id
+data_id, _ = project_client.upload_file("<data_file.jsonl>")
+```
+
+##### [Optional] Azure OpenAI Model
+
+Azure AI project comes with a default Azure Open AI endpoint which can be easily accessed using following code. This gives you the endpoint details for you Azure OpenAI endpoint. Some of evaluators need model that supports chat completion.
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:767]: Azure AI Studio project

Comment (dargilco) [inline sdk/ai/azure-ai-projects/README.md:767]: Some of the evaluators...

**Review by dargilco — APPROVED** [review]

File: sdk/ai/azure-ai-projects/README.md (hunk 4 above)
Comment (singankit) [reply sdk/ai/azure-ai-projects/README.md:738]: Right now we do not support that. Once we add support will make an update.

File: sdk/ai/azure-ai-projects/README.md [hunk 9]
```diff
@@ -718,6 +718,119 @@ print("Deleted agent")
 
 <!-- END SNIPPET -->
 
+### Evaluation
+
+Evaluation in Azure AI Project client library is desinged to assess the performance of generative AI applications in the cloud. The output of Generative AI application is quantitively measured with mathematical based metrics, AI-assisted quality and safety metrics. Metrics are defined as evaluators. Built-in or custom evaluators can provide comprehensive insights into the application's capabilities and limitations.
```
Comment (jhakulin) [inline sdk/ai/azure-ai-projects/README.md:723]: typo, desinged

File: sdk/ai/azure-ai-projects/README.md [hunk 10]
```diff
@@ -718,6 +718,119 @@ print("Deleted agent")
...
+
+#### Evaluator
+
+Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models or generative AI applications.
+
+Evaluators are made available via [azure-ai-evalaution][azure_ai_evaluation] SDK for local experience and also in [Evaluator Library][evaluator_library] in Azure AI Studio for using them in the cloud.
```
Comment (jhakulin) [inline sdk/ai/azure-ai-projects/README.md:729]: typo evalaution

File: sdk/ai/azure-ai-projects/README.md [hunk 11]
```diff
@@ -718,6 +718,119 @@ print("Deleted agent")
...
+- [Optional] Azure Open AI model.
+
+##### Evaluators
+
+For running evaluator in cloud, evaluator `ID` is needed. To get it via code you use [azure-ai-evaluation][azure_ai_evaluation]
+
+```
+# pip install azure-ai-evaluation
+
+from azure.ai.evaluation import RelevanceEvalautor
```
[the remark this answers is not in the document]
Comment (singankit) [reply sdk/ai/azure-ai-projects/README.md]: Thanks for catching it

**Pull request merged**